### ISSUE TITLE
Implement automatic import of projects

### ISSUE THREAD

**Issue by yyoncho** [emacs-lsp/lsp-java#39]
Implement automatic import of projects

**Comment by gregnwosu** [comment]
taken from the instructions relies on wget, whaddya think?

```elisp


(defun install-java-server ()
  "installs the java server"
  (interactive "P")
  (let  ((shell-cmd "rm -rf ~/.emacs.d/eclipse.jdt.ls/server/ && mkdir -p ~/.emacs.d/eclipse.jdt.ls/server/ && wget http://download.eclipse.org/jdtls/snapshots/jdt-language-server-latest.tar.gz -O /tmp/jdt-latest.tar && tar xf /tmp/jdt-latest.tar -C ~/.emacs.d/eclipse.jdt.ls/server/"))
(shell-command shell-cmd nil "lsp-install-err")))


```

**Comment by yyoncho** [comment]
I guess the comment is for: https://github.com/emacs-lsp/lsp-java/issues/40 ? 
As you have mentioned, the solution depends on `wget`, `rm`, and `untar` so it won't work on Windows unconditionally. The `jdt-ls` server is available through maven, and `dependency:unpack` can handle tar files so this is one of the possible implementations.

**Comment by yyoncho** [comment]
Pending fix at https://github.com/emacs-lsp/lsp-mode/pull/409

**Comment by yyoncho** [comment]
Fixed with  emacs-lsp/lsp-mode#409 . Now you can use lsp-workspace-folders-add/remove/switch .
